.gitattributes

```diff
@@ -19,6 +19,7 @@ src/data/*.json export-ignore
 src/data/*/*/*.json export-ignore
 src/data/*/*/docs*.json.php export-ignore
 src/data/*/*/examples*.json.php export-ignore
+src/data/*/*/endpoint-tests-*.php export-ignore
```

Comment:
`endpoint-tests-*.json` should be covered by line 19. We could probably leave this, but makes sense to change to match docs and examples above